tests/test_config.py

```diff
@@ -214,10 +208,50 @@ def test_entity_customization(self):
 
         self.hass.block_till_done()
 
-        state = self.hass.states.get('test.test')
+        return self.hass.states.get('test.test')
+
+    def test_entity_customization_false(self):
```

Comment:
This tests the schema as much as it test the customization.
For exampe the string splitting and lowercasing happens in `config.py`

All the code in `helpers/customization.py` is covered by its own tests.